src/Entity/Package.php

```diff
@@ -33,6 +33,33 @@
 use Composer\Util\HttpDownloader;
 use DateTimeInterface;
 
+enum PackageFreezeReason: string
+{
+    case Spam = 'spam';
+    case RemoteIdMismatch = 'remote_id';
+    case Gone = 'gone';
+
+    public function translationKey(): string
+    {
+        return 'freezing_reasons.' . $this->value;
```

Comment:
if you want to make those translations extractable, a solution would be to make a method returning a TranslatableMessage object (that you would still use in Twig by passing it to the `trans` filter), using a `match()` statement instead of concatenation (as dynamic keys cannot be extracted by the tooling)

Reply:
Thanks that's good to know but I don't really care about extractability here.